app/views/layouts/_navigation.html.erb

```diff
@@ -36,6 +36,9 @@
               <% if can?(:read, ToolType) %>
                 <li><%= link_to "Tool Types", tool_types_path %></li>
               <% end %>
+              <% if is_admin? %>
```

Comment:
Totally makes sense to keep the block in ApplicationController as long as it's using CanCan